src/main/scala/edu/ie3/simona/model/participant/evcs/EvcsModel.scala

```diff
@@ -500,20 +503,20 @@ final case class EvcsModel(
   ): Power = {
```

Comment:
Not sure, but wouldn't we expect ApparentPower for `getMaxAvailableChargingPower`?

Reply:
Ok I see the problem. I agree that normally the power factor for charging evs will be 1.0. But is this always the case? Have to think longer about this

Reply:
See my comment above

Reply:
See my other comment. tldr: Evcs = ApparentPower incl. qCharacteristics, EV = active power

This may will open new type annotation issues, I guess

Reply:
We only calculate active power in EvcsModel ourselves, reactive power is calculated in the background via qControl, like for all other SystemParticipants